src/lib.rs

```diff
@@ -273,13 +280,15 @@ pub fn pick_bkcolor(x: i32, y: i32) -> ColorT {
 	}
 }
 
-pub fn print(x: i32, y: i32, value: &str) -> i32 {
+pub fn print(x: i32, y: i32, value: &str) {
 	assert!(x >= 0);
 	assert!(y >= 0);
 
 	with_utf8_ptr(value, |ptr| {
 		unsafe {
-			terminal_print8(x, y, ptr)
+			let mut out_w = 0;
+			let mut out_h = 0;
+			terminal_print_ext8(x, y, 0, 0, TK_ALIGN_DEFAULT, ptr, &mut out_w, &mut out_h)
```

Comment:
Would you prefer I amend the commit and force push/create a new branch, or just create a new commit for this?